extras/rya.streams/api/src/test/java/org/apache/rya/streams/api/queries/InMemoryQueryRepositoryTest.java

```diff
@@ -167,7 +167,7 @@ public void updateListenerNotify() throws Exception {
 
             final Set<StreamsQuery> existing = queries.subscribe(new QueryChangeLogListener() {
                 @Override
-                public void notify(final ChangeLogEntry<QueryChange> queryChangeEvent) {
+                public void notify(final ChangeLogEntry<QueryChange> queryChangeEvent, final Optional<StreamsQuery> newQueryState) {
```

Comment:
The tests in this class need to be updated to ensure they are notified with the correct newQueryState value.

Reply:
This comment has not been addressed.